application/aam-backend-service/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM amazoncorretto:17-alpine
+COPY ./build/install/aam-backend-service /aam-backend-service
```

Comment:
Let's rename service and folder to something that includes "query" or "reporting" backend.